src/bp/core/services/middleware/event-engine.ts

```diff
@@ -119,10 +119,10 @@ export class EventEngine {
 
```

Comment:
Just curious to know if some issue prompted adding these awaits or they were just forgotten before, and how much it impacts perfromances on heavy loads